intranet/static/js/eighth/signup.search.js

```diff
@@ -215,11 +211,13 @@ $(document).ready(function() {
                 /* imply AND */
                 show = true;
                 for (i in shows) {
-                    if (!shows[i]) {show = false;}
+                    if (!shows[i]) show = false;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["i" used outside of binding context.](https://www.codacy.com/app/tjhsst-ion/ion/file/2743306435/issues/source?bid=3212528&fileBranchId=3355681#l214)